test/ci-test-weather.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Not really. It largely depends on how the schema are created and which are the dependencies.